### PR TITLE
update import in doc.go of staging/src/k8s.io/*

### PR DESCRIPTION
**Pull request by mysunshine92**
**What type of PR is this?**
> Uncomment only one, leave it on its own line:

 /kind documentation

**What this PR does / why we need it**:

update import in doc.go of staging/src/k8s.io/code-generator/
update import in doc.go of staging/src/k8s.io/csi-api/ 
update import in doc.go of staging/src/k8s.io/apiextensions-apiserver/ 
update import in doc.go of staging/src/k8s.io/sample-controller/

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Only a partial fix for #68231.

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by liggitt** [comment]
thanks for the PR, please combine with the updates in https://github.com/kubernetes/kubernetes/pull/71389 and https://github.com/kubernetes/kubernetes/pull/71386

**Comment by mysunshine92** [comment]
@liggitt thank you, I have combined this PR with the updates in #71389 and #71386

File: staging/src/k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset/doc.go [hunk 1]
```diff
@@ -17,4 +17,4 @@ limitations under the License.
 // Code generated by client-gen. DO NOT EDIT.
 
 // This package has the automatically generated clientset.
-package clientset
+package clientset // import "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset"
```
Comment (liggitt) [inline staging/src/k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset/doc.go]: these files are generated... unless the generator is updated as well, the scripts that verify the files match generated output will fail

Comment (mysunshine92) [reply staging/src/k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset/doc.go]: @liggitt ok, thank you.

**Comment by liggitt** [comment]
/close

the lines that this is modifying are produced by client-gen (https://github.com/kubernetes/code-generator/tree/master/cmd/client-gen). changes to the produced output need to be made there, then imported here and regenerated

**Pull request closed without merging**